### PR TITLE
Added option for uriQuery in query method

### PR DESCRIPTION
**Pull request by TomLous**
I wanted to supply preference information into the rest query
The runEsCommand allowed for this, but the parameter was not implemented in the overal query call

See:
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-preference.html

### CODE REVIEW

File: elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala [hunk 1]
```diff
@@ -75,9 +75,9 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
   import RestlasticSearchClient.ReturnTypes._
 
   def ready: Boolean = endpointProvider.ready
-  def query(index: Index, tpe: Type, query: QueryRootLike, rawJsonStr: Boolean = true): Future[SearchResponse] = {
+  def query(index: Index, tpe: Type, query: QueryRootLike, rawJsonStr: Boolean = true, uriQuery: UriQuery = UriQuery.Empty): Future[SearchResponse] = {
     implicit val ec = searchExecutionCtx
-    runEsCommand(query, s"/${index.name}/${tpe.name}/_search").map { rawJson =>
+    runEsCommand(query, s"/${index.name}/${tpe.name}/_search", query=uriQuery).map { rawJson =>
```
Comment (rcoh) [inline elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala:80]: Space around `=`

**Review by rcoh — CHANGES_REQUESTED** [review]
Please revert the pom changes

**Comment by TomLous** [comment]
I'm sorry the last commit should not be part of the pull request (I'm new at this)
How can I remove this from it?

**Comment by rcoh** [comment]
Probably easiest is to run
```
git revert 221b2d64f2a12cfbab011bfd966afd1ff2c9694b
git push
```

You can also rebase to remove it but that's a bit more complex.

**Comment by TomLous** [comment]
'fixed' it. Sorry for the mess.

**Review by rcoh — APPROVED** [review]

**Comment by rcoh** [comment]
Thanks for your contribution!

**Pull request merged**